chocolatey-community/chocolatey-packages
# (librecad) Switch to using new GitHub API helper

## Description

This pull request updates the automatic updater to make use of our new GitHub API helper to get the releases available.
This allows this package to once again be pushed to Chocolatey Community Repository.

Additionally, a small change to include the already pushed versions of 2.1 and 2.0 has been added, without having these added and if we need to create a fix version we would first need to add these manually or explicitly set the version to use.

fixes #2194

## Motivation and Context

To make the automatic updater work again.

## How Has this Been Tested?

1. Run `.\update_all.ps1 librecad`
2. Verify a package was created for 2.2.0
3. Run `.\update_all.ps1 librecad -ForcedPackages librecad\2.1`
4. Verify a fix version for version 2.1.3 was created

## Screenshot (if appropriate, usually isn't needed):

## Types of changes


## Checklist:


- [ ] The added/modified package passed install/uninstall in the chocolatey test environment (only update.ps1 is affected here, so install/uninstall has not been checked).
